jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/openshift/ImageChangeTriggerEnricher.java

```diff
@@ -111,9 +111,9 @@ public void visit(DeploymentConfigSpecBuilder builder) {
         });
     }
 
-    private Boolean isImageChangeTriggerNeeded(String containerName) {
+    private boolean isImageChangeTriggerNeeded(String containerName) {
         String containersFromConfig = Configs.asString(getConfig(Config.CONTAINERS));
-        Boolean enrichAll = getValueFromConfig(ENRICH_ALL_WITH_IMAGE_TRIGGERS, false);
+        boolean enrichAll = getValueFromConfig(ENRICH_ALL_WITH_IMAGE_TRIGGERS, false);
         JKubeBuildStrategy buildStrategy = getContext().getConfiguration().getJKubeBuildStrategy();
 
         if (Boolean.TRUE.equals(enrichAll)) {
```

Comment:
I think this should not b needed
```suggestion
        if (enrichAll) {
```